texts/formatted_evolution.txt

```diff
@@ -0,0 +1,1457 @@
+"<Evolution>"
+	"evolution" n sg
+"<is>"
+	"be" vbser pres p3 sg
+"<a>"
+	"a" det ind sg
+"<scientific>"
+	"scientific" adj
+"<theory>"
+	"theory" n sg
+"<used>"
+	"use" vblex past
+;	"use" vblex pp
+"<by>"
+;	"by" adv
+	"by" pr
+"<biologists>"
+	"biologist" n pl
+"<.>"
+	"." sent
+"<It>"
+	"prpers" prn subj p3 nt sg
+;	"prpers" prn obj p3 nt sg
+"<explains>"
+	"explain" vblex pres p3 sg
+"<how>"
+	"how" adv
+;	"how" preadv
+;	"how" adv itg
+"<living>"
+;	"live" vblex pprs
+;	"live" vblex ger
+;	"live" vblex subs
+	"living" adj
+;	"living" n sg
+"<things>"
+	"thing" n pl
+"<change>"
+;	"change" n sg
+;	"change" vblex inf
+	"change" vblex pres
+;	"change" vblex imp
+"<over>"
+;	"over" adv
+	"over" pr
+"<a>"
+	"a" det ind sg
+"<long>"
+;	"long" adv
+	"long" adj sint
+;	"long" vblex inf
+;	"long" vblex pres
+;	"long" vblex imp
+"<time>"
+	"time" n sg
+;	"time" vblex inf
+;	"time" vblex pres
+;	"time" vblex imp
+"<,>"
+	"," cm
+"<and>"
+	"and" cnjcoo
+"<how>"
+	"how" adv
+;	"how" preadv
+;	"how" adv itg
+"<they>"
+	"prpers" prn subj p3 mf pl
+"<have>"
+;	"have" vbhaver inf
+	"have" vbhaver pres
+;	"have" vblex inf
+;	"have" vblex pres
+;	"have" vblex imp
+"<come to>"
+	"come# to" vblex inf
+;	"come# to" vblex pres
+;	"come# to" vblex pp
+;	"come# to" vblex imp
+"<be>"
+	"be" vbser inf
+"<the>"
+	"the" det def sp
+"<way>"
+	"way" n sg
+"<they>"
+	"prpers" prn subj p3 mf pl
+"<are>"
+;	"be" vbser pres
+;	"be" vbser pres p1 pl
+;	"be" vbser pres p2 sg
+;	"be" vbser pres p2 pl
+	"be" vbser pres p3 pl
+
+"<Earth>"
+	"earth" n sg
+"<is>"
+	"be" vbser pres p3 sg
+"<very>"
+	"very" adv
+;	"very" preadv
+"<old>"
+	"old" adj sint
+"<.>"
+	"." sent
+"<By>"
+;	"by" adv
+	"by" pr
+"<studying>"
+;	"study" vblex pprs
+	"study" vblex ger
```

Comment:
I don't think it's a gerund, could you clarify on this? Also, not sure what subs is supposed to be?

Reply:
Examples of similar gerund usage: https://portlandenglish.edu/blog/by-gerund/